bin/resources/GameIndex.yaml

```diff
@@ -11251,6 +11251,8 @@ SLES-51758:
   name: "Metal Arms - Glitch in the System"
   region: "PAL-M5"
   compat: 5
+  speedHacks:
+    mvuFlagSpeedHack: 0  # Fixes SPS.
```

Comment:
can you get rid of the extra space after the 0 please? should only be one space (I know it's a bit inconsistent in the file but I'm trying to rectify that)

Reply:
Fixed.